openshift/templates/issuer-web/issuer-web-deploy.a2a.param

```diff
@@ -22,7 +22,7 @@ APP_LOGO_FILE_NAME=logo.svg
 APP_TERMS_FILE_NAME=terms-and-conditions.html
 APP_UNAUTHORIZED_FILE_NAME=unauthorized.html
 APP_VUETIFY_FILE_NAME=vuetify.json
-WEB_HOST_NAME=
+WEB_HOST=
```

Comment:
`WEB_HOST` should remain the same OR you should also update the Caddyfile(s) to pick up the new variable name - I would leave `WEB_HOST` in this specific case, it is not used in the diagram annotations anyway.

Applies to other files as well.